ui/apps/platform/src/Containers/Login/LoginPage.js

```diff
@@ -174,6 +174,19 @@ class LoginPage extends Component {
         if (authorizeRoxctlMode) {
             authProviders = authProviders.filter((provider) => provider.type !== 'basic');
             title = 'Authorize roxctl';
+            if (authProviders?.length === 0) {
```

Comment:
Question that is not a blocker to merge: is `?` optional chaining needed here?

It seems like `authProviders.filter(…)` call above would throw if the value is nullish.